e2e/cli-e2e/tests/help.e2e.test.ts

```diff
@@ -18,7 +18,7 @@ describe('CLI help', () => {
     });
     const helpCommandResult = await executeProcess({
       command: 'code-pushup',
-      args: ['--help'],
+      args: ['help'],
```

Comment:
The point of this test is to check that `--help` has the same output as `help`. But now you're comparing `help` with `help` :sweat_smile:

Reply:
reverted